tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+	"include": [
+		"js"
+	],
+	"compilerOptions": {
```

Comment:
Add `downlevelIteration: true`, since we're using the `for..of` on the model-view Maps. 

https://www.typescriptlang.org/tsconfig/#downlevelIteration

Reply:
Cool, thanks!